app/src/main/java/com/android/doctorapp/ui/userdashboard/userfragment/UserAppointmentFragment.kt

```diff
@@ -70,6 +73,17 @@ class UserAppointmentFragment :
         savedInstanceState: Bundle?
     ): View {
         super.onCreateView(inflater, container, savedInstanceState)
+        lifecycleScope.launch {
+            viewModel.session.getBoolean(IS_ENABLED_DARK_MODE).collectLatest {
+                if (it == true)
```

Comment:
can you please check why we need to put if(it ==true ). As per my understanding if you store boolean in shared Preferences then no need to check ==true. Directly put in **it** in if condition.

Please check in everyWhere where should u used this kind of condition.